tests/test_list_all_links.py

```diff
@@ -78,7 +91,9 @@ def test_all_links_wrong_method_body():
     v = Validator(
         {
             'status': {'type': 'integer', 'allowed': [405]},
-            'message': {'type': 'string', 'allowed': ['Method Not Allowed']}
+            'message': {
+                'type': 'string', 'allowed': ['Method Not Allowed']
+            }
         }
     )
     assert v.validate(response.json())
```

Comment:
I think you should make this assert more verbose. 
`assert v.validate(response.json()), v.errors`